layouts/partials/footer.html

```diff
@@ -19,11 +19,11 @@
       <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
         {{ with .Site.Params.copyright }}<small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>{{ end }}
 		  <br>
-
-		  {{ with .Site.Params.terms_and_conditions }}<small class="ml-1"><a href="{{ . }}" target="_blank">{{ T "footer_terms_and_conditions" }}</a></small>{{ end }}
-
-		  {{ with .Site.Params.terms_of_service }}<small class="ml-1"><a href="{{ . }}" target="_blank">{{ T "footer_terms_of_service" }}</a></small>{{ end }}
-
+      <small class="text-white ml-1">Use of Armory products and features are subject to the
```

Comment:
Small grammar nit: 
```suggestion
      <small class="text-white ml-1">Use of Armory products and features is subject to the
```
"Use" is singular...